src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs

```diff
@@ -61,14 +61,14 @@ internal class RavenQueryProviderProcessor<T>
             "AS",
             "SELECT",
             "WHERE",
-            "LOAD",
```

Comment:
why did you remove it?

Reply:
because load is also reserved in js

Reply:
but now i realize that this query won't work because of it:
from o in session.Query<Order>() select new { Load = o.Company };